src/app/components/AllowanceMenu/index.tsx

```diff
@@ -34,7 +34,7 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
   async function updateAllowance() {
     await utils.call("updateAllowance", {
       id: parseInt(allowance.id),
-      totalBudget: budget,
+      totalBudget: parseInt(budget),
```

Comment:
Is it still a thing to add the radix? I remember not adding radix used to cause issues in certain situations:  
https://eslint.org/docs/rules/radix

Reply:
Our ECMAScript target version is ES6 so this shouldn't be a problem.